operations-stdlib/src/commonTest/kotlin/string/CompareToDateTest.kt

```diff
@@ -0,0 +1,71 @@
+package string
+
+import JsonLogicResult.Failure
+import JsonLogicResult.Success
+import JsonLogicEngine
+import TestInput
+import io.kotest.core.spec.style.FunSpec
+import io.kotest.datatest.withData
+import io.kotest.matchers.shouldBe
+import string.CompareToDate
+
+class CompareToDateTest:FunSpec({
+    val operatorName = "compareToDate"
+    val logicEngine = JsonLogicEngine.Builder().addStandardOperations(
+        mapOf(
+            operatorName to CompareToDate
+        )
+    ).build()
+    withData(
+        nameFn = { input -> "Should evaluated ${input.expression} with given ${input.data} result in ${input.result}"},
```

Comment:
more edgy cases will be beneficial. Ania mentioned about exceeded date range in February which passes the flow :/ Please provide more tests with cases where input is unwrapped for example wrong arguments order etc.

Reply:
Maybe instead of covering all edge cases by yourself just use this library?
https://github.com/Kotlin/kotlinx-datetime
`LocalDateTime(year, month, day, 0, 0, 0, 0)` is created only when year, month and day are valid (and there is `compareTo` function available)

Reply:
`LocalDate` IMO will be enough in our case. I added this library. Please check and verify a way to add it to the project. 
All unnecessary files and code should be removed right now.